router/broker.go

```diff
@@ -33,12 +35,38 @@ type subscription struct {
 	subscribers map[*wamp.Session]struct{}
 }
 
+// storedEvent is a wrapper around wamp event message with timestamp
+// to be used in event history
+type storedEvent struct {
```

Comment:
Could you embed this inside `eventHistoryItem` and drop the separate type altogether or do you need to express an event that's stored separately for two different publications?

If so perhaps you could store all of that plus `[]wamp.Publish` as a publication slice.

Having three types (collection, pointer-to-entry-and-publication and entry) is more confusing than just two (collection and entry)